app/src/main/kotlin/com/hippo/ehviewer/ui/MainActivity.kt

```diff
@@ -362,10 +363,10 @@ class MainActivity : EhActivity() {
                     SideDrawer(
                         drawerContent = {
                             if (sheet != null) {
+                                val insets = WindowInsets.systemBars.only(WindowInsetsSides.Top + WindowInsetsSides.End)
                                 ModalDrawerSheet(
-                                    modifier = Modifier.widthIn(max = (configuration.screenWidthDp - 112).dp),
+                                    modifier = Modifier.widthIn(max = (configuration.screenWidthDp - 112).dp).windowInsetsPadding(insets),
                                     drawerShape = ShapeDefaults.Large.copy(topEnd = CornerSize(0), bottomEnd = CornerSize(0)),
-                                    windowInsets = WindowInsets.systemBars.only(WindowInsetsSides.Top + WindowInsetsSides.End),
```

Comment:
This ain't the way.
`ModalDrawerSheet` takes vertical system bars insets by default and pad content for them.
If you want side sheet to be edge-to-edge, you should handle the insets in the lazy list;
if you don't, consume bottom insets as well in windowInsetsPadding.

Reply:
Ah, I think we should limit SideSheet height as well
2/3 screen width?

Reply:
> Ah, I think we should limit SideSheet height as well 2/3 screen height?

Why? I think side sheet height is 100% in M3 specs.